core/src/navigation/GlobalSearch.html

```diff
@@ -181,12 +212,22 @@
         }
       } else if (GenericHelpers.isFunction(search.searchProvider.onInput)) {
         search.searchProvider.onInput();
+        if (search.searchFieldCentered) {
+          document.querySelector('.luigi-search__input').value
+            ? (clearSearchFieldBtn = true)
+            : (clearSearchFieldBtn = false);
```

Comment:
I did not get this logic. Did you want to clear the search input value?
Could you please briefly explain what should be done here?

Reply:
Showing the clear button if input field has some value